monitoring/README.md

```diff
@@ -0,0 +1,48 @@
+# Monitoring
+This directory provides tools to monitor cdc using prometheus to scrape the metrics and grafana to plot graphs on the collected metric data.
+We first create docker containers for Prometheus, Grafana and Kafka Connect. Then we connect
+Yugabyte DB to Kafka-connect as a source connector.
+
+## Pre-requisites
+Ensure that docker and docker compose is installed.
+We will need an instance of YugabyteDB running (see
+https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-basics/create-clusters/) with a stream created on it.
```

Comment:
Use [markdown links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)